
Patch from: https://github.com/google/packetdrill/pull/93
File paths fixed for working directory.

From 528c51a5c8663116e1b8e5be85a9c461f9778c20 Mon Sep 17 00:00:00 2001
From: Yaakov Selkowitz <yselkowi@redhat.com>
Date: Fri, 21 Mar 2025 17:09:58 -0400
Subject: [PATCH] Use standard bool type

GCC 15 defaults to C23, in which bool, true, and false are keywords.
Using <stdbool.h> keeps this compatible with older C standards.

Signed-off-by: Yaakov Selkowitz <yselkowi@redhat.com>
---
 gtests/net/packetdrill/types.h | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/gtests/net/packetdrill/types.h b/gtests/net/packetdrill/types.h
index 36571fb3..8afbd803 100644
--- a/types.h
+++ b/types.h
@@ -32,6 +32,7 @@
  */
 #define _GNU_SOURCE		/* for asprintf */
 #include <limits.h>
+#include <stdbool.h>
 #include <stdio.h>
 #include <string.h>
 #include <arpa/inet.h>
@@ -61,12 +62,6 @@ typedef signed short s16;
 typedef signed int s32;
 typedef signed long long s64;
 
-typedef u8 bool;
-enum bool_t {
-	false = 0,
-	true = 1,
-};
-
 #define ARRAY_SIZE(array_name)  (sizeof(array_name) / sizeof(array_name[0]))
 #define TIMEOUT_NONE -1
 
